Harmonize button spacer and remove duplicate delete button in drawing

ltshb opened the pull request:
Use a global variable to add space between adjacent buttons.

Also removed useless legacy css code in FeatureStyleEdit.

[Test link](https://sys-map.dev.bgdi.ch/feat-design-tune/index.html)

pakb commented on src/modules/infobox/components/FeatureProfile.vue:
```diff
@@ -53,11 +53,6 @@
                 data-cy="profile-popup-csv-download-button"
                 @click="onCSVDownload"
             />
-            <ButtonWithIcon
-                :button-font-awesome-icon="['far', 'trash-alt']"
-                data-cy="profile-popup-delete-button"
-                @click="onDelete"
-            />
```
I think this should stay. How are we supposed to remove a measure line without this button?

ltshb replied:
@pakb See screenshot below, this line remove the button crossed in red, but you still have the button in the description popup to delete the line. I don't think we need 2 buttons here.
![Uploading profile-delete-button.png…]()

pakb replied:
that works for a line yes, and it makes sense to remove the duplicate in this case, but what about measure?
![image](https://user-images.githubusercontent.com/11540521/199476485-585b4ffb-b82d-4504-b882-a1ad9dca72ba.png)

ltshb replied:
Very good point ! I didn't know about this use case. So I'll check if I can easily remove only for the line otherwise I leave it.

pakb reviewed (commented):
Just some doubts about the delete button removal, otherwise nice looking!

davidoesch commented:
Hmm once you are in measure mode
![image](https://user-images.githubusercontent.com/4577727/199485191-ee158753-7c59-43a0-9ae2-5fdb9e22070b.png)
 there is no pop up. the delete button was for : save the user the step for closing the reslt window and klick again on the line to delete it.... saves saves the user three klicks. 

the measurement function does not have a delete option... thsi is wy we added the delete button

even if it is a button more ...I would keep it in this case

ltshb commented:
@davidoesch @pakb I reworked the PR. Now in the profile the delete button is only shown for the measure tool and not for the line polygon. For the later the delete button is part of the description popup. This way for line polygon we only have 1 delete button instead of two. This way we save place on mobile device for the profile infos.
For a better nice looking I also added a small margin between buttons.
